pagerduty/resource_pagerduty_event_orchestration.go

```diff
@@ -88,6 +88,11 @@ func buildEventOrchestrationStruct(d *schema.ResourceData) *pagerduty.EventOrche
 
 	if attr, ok := d.GetOk("team"); ok {
 		orchestration.Team = expandOrchestrationTeam(attr)
+	} else {
+		var tId *string = nil
+		orchestration.Team = &pagerduty.EventOrchestrationObject{
+			ID: tId,
+		}
```

Comment:
This is the main change - we are now explicitly setting the team ID to `nil` if it's not present on the resource

Reply:
Question because I'm a noob, why can't it just be `var tId *string` aren't string pointers by default nil?

Reply:
Very good question, let me test it and see if we can just declare it :)

Reply:
@tyurina I think you're right, we don't need to instantiate the variable. What I noticed is that Step 2 in the test sporadically fails with an "invalid team id" message., possibly because the team is created right before the orchestration. I think I may need to add 1 retry into the update method to fix it.